pkgs/native_assets_builder/lib/src/build_runner/build_runner.dart

```diff
@@ -291,6 +290,21 @@ class NativeAssetsBuildRunner {
         if (dependencies.contains(entry.key)) entry.key: entry.value,
     };
   }
+
+  void validateAssetsPackage(List<Asset> assets, String packageName) {
+    final invalidAssetIds = assets
+        .map((a) => a.name)
+        .where((n) => !n.startsWith('package:$packageName'))
```

Comment:
also here should be `'package:$packageName/'`.

Reply:
Done.